Ramos_HwAssignments/Ramos_Wk4_WeatherApp/Forecast.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState, useEffect } from 'react';
+import { NavigationContainer, useNavigation, useRoute, RouteProp, StackActions } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import { Text, View, SafeAreaView, TextInput, Alert, FlatList, TouchableOpacity } from 'react-native';
+import { StyleSheet } from 'react-native';
+import { StackParamList } from './App';
+import { useFetch } from './Functions/useFetch';
+import { PhotoItem } from './PhotoItem';
+
+type WeatherNavigationProp = StackNavigationProp<StackParamList, 'Weather'>;
+
+export const Forecast = () => {
+  const navigation = useNavigation<WeatherNavigationProp>();
+
+  const route = useRoute();
+  const { timeSpan } = route.params;
+
+  const {data, loading, error} = useFetch(`http://api.weatherapi.com/v1/forecast.json?q=02893&days=${timeSpan}&key=2c1c6cbbec964d26907230624231608`);
+
+  const [location, setLocation] = useState('');
+  const [currentWeather, setCurrentWeather] =  useState('');
```

Comment:
Again no need for separate state or useEffect here because it's already being done in `useFetch`